### PR TITLE
loose catbeasts are now black

### PR DESCRIPTION
**Pull request by Intigracy**
human_face.dmi is edited because I had to cut the ears off of the hair and make them part of the head, unless you want brown ears on black beasts.
A side effect of this is we're probably going to see some ears sticking out of helmets that they're using to try to hide like little babies. A good change in my opinion!

There's a few more files changed that have to do with the way the icon-based skin colors work, because it is/was a mess that still needs further work (random skin tone should be moved to species datums, for example)

:cl:
 * tweak: Loose catbeasts are now black.

### CODE REVIEW

**Comment by MadmanMartian** [comment]
>RUNTIME IS ROGUE

**Comment by DeityLink** [comment]
So it's like because they're bad omen or something?

File: code/__HELPERS/mobs.dm [hunk 1]
```diff
@@ -93,18 +95,24 @@ proc/skintone2racedescription(tone, species = "Human")
...
-			if(2)
-				return "brown"
-			if(3)
+			if(VOXGRAY)
 				return "gray"
+			if(VOXBROWN)
+				return "brown"
 			else
 				return "green"
+	else if(species == "Tajaran"
```
Comment (ShiftyRail) [inline code/__HELPERS/mobs.dm]: You forgot a `)`

**Comment by boy2mantwicethefam** [comment]
That's racist you can't say the n-word

**Comment by Kurfursten** [comment]
I feel like a better implementation of this would be to create a black hairstyle and a brown hairstyle and just set which one they get based on if they're black or not.

**Comment by Hoshi-chan** [comment]
13% of all catbeasts...

**Comment by SonixApache** [comment]
when

**Comment by Kurfursten** [comment]
If you're gonna do it, do it right. We both know this isn't the right way to do it. Make it a black hairstyle and set h_style then call update_hair.

Editing the face file is a hacky solution.

**Review by Kurfursten — CHANGES_REQUESTED** [review]
If you're gonna do it, do it right. We both know this isn't the right way to do it. Make it a second racial hairstyle and set h_style then call update_hair.

Editing the face file is a hacky solution.

**Comment by Intigracy** [comment]
Nope.

I'd rather see the ears poking out of things.

**Comment by Intigracy** [comment]
You can merge it or close it, but I have no interest in changing it.

**Comment by Zth--** [comment]
Uhm, closing the PR over personal preference? ... when there's clearly more active collabs?

This is not how I want my collabs to behave

**Comment by Kurfursten** [comment]
This isn't personal preference, this is a matter of coding standards. In the original comment he claimed it was only this way because he didn't know how to manage it besides this hacky solution, and now he's gone back to change it to say he prefers it this way. It's not changeloged, it's not atomic, and the last comment gives off a serious air of "I'm too lazy to do this right".

And not that it matters but I am currently the most active collab on this repo according to Insights. Nor does this matter, but I'm going to take the sprites and do this the right way instead of half assing it.

**Comment by Intigracy** [comment]
Nah I'll just make a poll to see if people want to see ears through
headgear.

**Comment by Kurfursten** [comment]
I feel like this is unnecessarily dramatic with the ultimatum. Talk to me on Discord and we'll find common ground.

There's are two reasons why it's designed the way it is: it's consistent with the tail behavior, and it creates an interesting gameplay scenario in which people are forced to remove helmets to identify catbeasts. Stopping unknowns for questioning. Designing the feature in the way you have is gunked because it encourages people to look for things with large on-character sprites to exploit how our clothing overlay code works. 

But *even if* people wanted cat ears visible at all times, this would still not be the proper implementation. You should still make it a hairstyle and add an exception so that it draws the ears on top of all helmets. Don't insist on code we both know is bad out of stubbornness. If you want help coding this I am happy to help you or, honestly, just do it myself.

**Comment by Intigracy** [comment]
Sorry, I'm being unfair here. Do what you think is the better method.

Though the difference in doing it as a hair vs part of the face is that the entire ear shows instead of just the part that would be sticking out from whatever headgear.

**Comment by Kurfursten** [comment]
Yeah, that's what I'm saying. I think if we *do* want ears sticking out, we should go full monte and have the whole ear sticking out. I think it's still fair to poll if you want to as long as people know the reasons it is how it is.

Because if that's the way people want it, it should apply evenly to all hats from big to small. The snakehat cosmetic shouldn't be better at hiding ears than an airtight space helmet, dig?

**Comment by Intigracy** [comment]
That looks absolutely awful though

**Comment by Kurfursten** [comment]
I mean I think ears-through-hardsuit-helmet is gonna look bad no matter how you do it

**Comment by Zth--** [comment]
As long as you reach consensus then it's fine by me. I think closing this PR was rather harsh and that's why I commented in the first place. But yeah let's get over this

**Pull request closed without merging**